projects/core/directives/dropdown/dropdown-position-sided.directive.ts

```diff
@@ -23,6 +23,9 @@ export class TuiDropdownPositionSidedDirective implements TuiPositionAccessor {
     @Input()
     tuiDropdownSided: boolean | string = ``;
 
+    @Input()
+    tuiDropdownSidedOffset = 4;
```

Comment:
do we need?

```ts
@Input()
tuiDropdownSidedOffset = this.options.offset;
```

Reply:
Let's leave it for now. It's a different offset and I don't want to create a new token just for that.